planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -709,9 +705,8 @@ LaneChangeTargetObjectIndices NormalLaneChange::filterObject(
     // check if the object intersects with target lanes
     if (target_polygon && boost::geometry::intersects(target_polygon.value(), obj_polygon)) {
       // ignore static parked object that are in front of the ego vehicle in target lanes
-      bool is_parked_object = utils::lane_change::isParkedObject(
-        target_path, route_handler, extended_object, object_check_min_road_shoulder_width,
-        object_shiftable_ratio_threshold);
+      bool is_parked_object =
```

Comment:
@rej55 
I guess you can also use this [funciton](https://github.com/autowarefoundation/autoware.universe/blob/c8ee0d4a37a541ee0f3e8e754f40689eff2ab628/planning/behavior_path_planner/include/behavior_path_planner/utils/path_safety_checker/objects_filtering.hpp/#L41-L55)  and this [function](https://github.com/autowarefoundation/autoware.universe/blob/c8ee0d4a37a541ee0f3e8e754f40689eff2ab628/planning/behavior_path_planner/include/behavior_path_planner/utils/path_safety_checker/objects_filtering.hpp/#L77-L90) for the same purpose (just for sharing info :+1: )